src/common/TALENTS/warrior.ts

```diff
@@ -337,6 +379,30 @@ const talents = createTalentList({
     icon: 'ability_creature_cursed_04',
     maxRanks: 1,
   },
+  WAR_MACHINE_TALENT: {
```

Comment:
This is the Arms version of the talent, as a note.

Reply:
oh no did war machine still end up weird, not like this